### DISCUSSION TITLE
Personal Access Token: Repo Admin but not Org Owner

### DISCUSSION THREAD

**Discussion by ax3l** [community/community#77447] [Apps, API and Webhooks] [answered]
### Select Topic Area

Question

### Body

Hi,

I would like to create a new, fine-grained personal access token (PAT, currently in beta) for a repository
  https://github.com/settings/personal-access-tokens/new

I noticed that I cannot select a repository for which I am *admin* but not org *owner*.

Is this intentional? Can support for such a repo be added?
Otherwise, I would have to be made co-owner on the whole org.

![image](https://github.com/community/community/assets/1353258/374f041d-641a-4358-b14b-1214658b674b)
Since I am not an `AMReX-Codes` org repo co-owner, I cannot select `AMReX-Codes` as the *resource owner*, but maybe I do not need to, because I am admin of the repo in question...?

Repo in question:
- https://github.com/AMReX-Codes/pyamrex

Org in question:
- https://github.com/AMReX-Codes

Thanks,
Axel

**Comment by ax3l** [comment, marked as answer]
Found it, we had to set up:
https://github.com/organizations/AMReX-Codes/settings/personal-access-tokens-onboarding

**Comment by queenofcorgis** [comment]
We’re glad you could get this solved and we want to make sure others can do the same so we’ve reopened this discussion for better discoverability for our members.
